numpy/core/src/multiarray/ctors.c

```diff
@@ -1802,7 +1808,17 @@ PyArray_GetArrayParamsFromObject(PyObject *op,
             return 0;
         }
         /* If object arrays are forced */
-        if (is_object) {
+        if (is_object != DISCOVERED_OK) {
+            if (is_object == DISCOVERED_RAGGED && requested_dtype == NULL) {
+                /* NumPy 1.18, 2019-11-01 */
```

Comment:
Should this be 19 now?

Reply:
fixing

Reply:
Looks like this was missed? I can fix it up though.